rialto_airflow/publish/orcid.py

```diff
@@ -34,7 +34,7 @@ def export_author_orcids(data_dir):
                 row_values = {
                     "sunetid": row["sunetid"],
                     "full_name": row["full_name"],
-                    "orcidid": row["orcidid"],
+                    "orcidid": row["orcidid"] if row["orcidid"].strip() != "" else None,
```

Comment:
in practice, when i queried stage, i only saw empty string (`"""`, and got no results for whitespace (e.g. `orcidid = ''` got results, `orcidid LIKE ' %'` didn't).  but the whitespace stripping felt like a harmless bit of extra protection.